### DISCUSSION TITLE
Option to exit dashboard

### DISCUSSION THREAD

**Discussion by jrschenk** [folke/snacks.nvim#211] [Q&A] [answered]
I want to add an option to exit the dashboard without doing anything and go straight into Neovim. Is there a function that I can call in  snacks.dashboard that will do this? I looked but I wasn't able to find anything, but admittedly I don't know much about Lua.

**Comment by SuXYIO** [comment]
What do you mean "go straight into Neovim"? Like go open a file or exit nvim entirely?

**Reply by jrschenk** [reply]
I mean bypass the dashboard and maybe do something else that is not one of the listed options in the dashboard.

**Comment by folke** [comment]
If you open the dashboard **after** you already used Neovim, you can press `<esc>` to close the dashboard.

**Reply by jrschenk** [reply]
I want to close the dashboard just after starting Neovim., ih which case <esc> doesn't seem to do anything.

**Reply by folke** [reply, marked as answer]
I don't understand what you want to achieve. `:enew`?

**Reply by jrschenk** [reply]
Thanks, it's sorted now.  Basically :enew works, except I don't  want a new file.

**Reply by folke** [reply]
And what exactly do you want then? I really don't understand what you want to achieve here?
Neovim can't exist without a buffer, so if you close the dashboard, then the only thing you can do is create a new empty buffer (enew)

**Reply by jrschenk** [reply]
Thanks Folke.  I switched from Alpha dashboard recently and I had set up something in Alpha that exited the dashboard, and  I didn't realize that it was creating a new buffer. I wanted the same in dashboard which I have now. I added an option to create a new buffer without going into insert mode.
